openmls/Cargo.toml

```diff
@@ -27,6 +27,7 @@ itertools = {version = "0.10", optional = true}
 default = ["rust-crypto"]
 rust-crypto = ["evercrypt/rust-crypto-aes"]
 expose-test-vectors = ["itertools"]
```

Comment:
Do you want to keep both feature flags? I thought `test-utils` could expose the test vectors as well.

Reply:
I thought that was what you were suggesting. I have no preference either way. Then I would propose we only keep the `expose-test-vector` for now or at least until we need to expose only the test_utils.

Reply:
Same as above, only that I actually changed the name everywhere to `test-utils`, so there's no more `expose-test-vectors`.